File: app/src/main/java/com/example/doan/Models/Store.java
package com.example.doan.Models;

import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class Store implements Serializable {

    @SerializedName("id")
    private int id;

    @SerializedName("storeName")
    private String storeName;

    @SerializedName("address")
    private String address;

    @SerializedName("latitude")
    private double latitude;

    @SerializedName("longitude")
    private double longitude;

    @SerializedName("openTime")
    private String openTime;

    @SerializedName("closeTime")
    private String closeTime;

    @SerializedName("phone")
    private String phone;

    @SerializedName("imageUrl")
    private String imageUrl;

    public Store() {}

    public Store(int id, String storeName, String address, double latitude, double longitude, String openTime, String closeTime, String phone, String imageUrl) {
        this.id = id;
        this.storeName = storeName;
        this.address = address;
        this.latitude = latitude;
        this.longitude = longitude;
        this.openTime = openTime;
        this.closeTime = closeTime;
        this.phone = phone;
        this.imageUrl = imageUrl;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getStoreName() {
        return storeName;
    }

    public void setStoreName(String storeName) {
        this.storeName = storeName;
    }

    public String getName() {
        return storeName;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    public String getOpenTime() {
        return openTime;
    }

    public void setOpenTime(String openTime) {
        this.openTime = openTime;
    }

    public String getCloseTime() {
        return closeTime;
    }

    public void setCloseTime(String closeTime) {
        this.closeTime = closeTime;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: app/src/main/java/com/example/doan/Models/Store.kt
package com.example.doan.Models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Store(
    @SerializedName("id")
    var id: Int = 0,
    
    @SerializedName("storeName")
    var storeName: String? = null,
    
    @SerializedName("address")
    var address: String? = null,
    
    @SerializedName("latitude")
    var latitude: Double = 0.0,
    
    @SerializedName("longitude")
    var longitude: Double = 0.0,
    
    @SerializedName("openTime")
    var openTime: String? = null,
    
    @SerializedName("closeTime")
    var closeTime: String? = null,
    
    @SerializedName("phone")
    var phone: String? = null,
    
    @SerializedName("imageUrl")
    var imageUrl: String? = null
) : Serializable {
    val name: String?
        get() = storeName
}
